lawnchair/src/app/lawnchair/preferences2/PreferenceManager2.kt

```diff
@@ -78,6 +78,12 @@ class PreferenceManager2(private val context: Context) : PreferenceManager {
         save = { it.toString() },
     )
 
+    val showNotificationCount = preference(
+        key = booleanPreferencesKey(name = "show_notification_count"),
+        defaultValue = context.resources.getBoolean(R.bool.config_default_show_notification_count),
+        onSet = { reloadHelper.restart() },
```

Comment:
Do we really need to restart here? A grid reload should be enough.

Reply:
My bad. My force push resolves this.